components/script/dom/webidls/OffscreenCanvasRenderingContext2d.webidl

```diff
@@ -0,0 +1,21 @@
+[Exposed=(Window,Worker), Pref="dom.offscreen_canvas.enabled"]
+interface OffscreenCanvasRenderingContext2D {
```

Comment:
TravisCI doesn't like that this file is named `OffscreenCanvasRenderingContext2d.webidl`, while the interface is named `OffscreenCanvasRenderingContext2D`. The filename should match the interface name (ie. `2D`, not `2d`).